haiku-misc/space/space-1.2.recipe

```diff
@@ -0,0 +1,43 @@
+SUMMARY="A space screensaver"
+DESCRIPTION="Space uses real math to give users an awesome, \
+smooth, stellar experience, in real perspective."
+HOMEPAGE="https://github.com/HaikuArchives/Space"
+COPYRIGHT="David Enderso"
+LICENSE="GNU GPL v2"
+REVISION="1"
+srcGitRev="80216ae2e0821d122f447a1671d790e41094ecc7"
+SOURCE_URI="https://github.com/HaikuArchives/space/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="9b432eb68515c4b85bf2d261d49f1840847bb91442fa446edf555493944aa8d3"
+SOURCE_FILENAME="Space-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="Space-$srcGitRev"
+
+ARCHITECTURES="x86_gcc2 x86_64"
```

Comment:
Thanks @bach5000 for your PR!
As expected, this screen saver builds and on all architectures, including x86, so you may add `x86` to `ARCHITECTURES`.
```diff
-ARCHITECTURES="x86_gcc2 x86_64"
+ARCHITECTURES="x86_gcc2 x86 x86_64"
```

Reply:
BTW, I wrote "may" instead of "should" because this is not required.
That said, adding a `x86` or even a `?x86` is more "inclusive", as it does not ignore those users who have 32-bit only hardware and build their Haiku/x86 system themselves.